source/js/components/petition/petition.jsx

```diff
@@ -292,12 +312,23 @@ export default class Petition extends React.Component {
    * @returns {jsx} What users see when their petition sign up succeeded.
    */
   renderThankYou() {
-    return (
-      <div>
-        <p>{this.props.thankYou}</p>
-        <a href={this.props.shareLink} className="btn btn-info">{this.props.shareText}</a>
-      </div>
-    );
+    if (this.props.shareLink) {
```

Comment:
This means existing petitions that have a share link and no button links will continue to work as-is?

Reply:
That's the idea, but I contemplated having an if for each type of value, and if non exist, we just don't see any buttons.

Reply:
let's file a follow-up issue to disable this field on the admin.

Reply:
I think the idea is we can use either or. This doesn't replace the previous functionality, just allows us to pick.

Or are you thinking a way to toggle (disable) the field on and off via the admin page?